roles/contiv_network/tasks/main.yml

```diff
@@ -1,44 +1,10 @@
 ---
 # This role contains tasks for configuring and starting netmaster and netplugin service
-
-- name: download netmaster and netplugin
-  get_url:
-    validate_certs: no
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-12-11-2015.20-54-40.UTC/netplugin-v0.0.0-12-11-2015.20-54-40.UTC.tar.bz2
-    dest: /tmp/contivnet.tar.bz2
-
-- name: install netmaster and netplugin
-  shell: tar vxjf /tmp/contivnet.tar.bz2
-  args:
-    chdir: /usr/bin/
-    creates: netmaster
-
-- name: copy environment file for netplugin
-  template: src=netplugin.j2 dest=/etc/default/netplugin
-
-- name: copy systemd units for netplugin
-  copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
-
-- name: start netplugin
-  shell: systemctl daemon-reload && systemctl start netplugin
-
-- name: setup netmaster host alias
-  shell: echo "{{ service_vip }} netmaster" >> /etc/hosts
-
-- name: copy environment file for netmaster
-  copy: src=netmaster dest=/etc/default/netmaster
-  when: run_as == "master"
-
-- name: copy systemd units for netmaster
-  copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
-  when: run_as == "master"
-
-# resolves an issue with netmaster: https://github.com/contiv/netplugin/issues/210
-- name: start netmaster
-  shell: systemctl daemon-reload && systemctl start netmaster
-  when: run_as == "master"
-  run_once: true
-
-# FIXME: need to move the following to correct roles -- also needs release tooling
-- name: install contivctl
-  shell: source /etc/profile.d/00golang.sh && go get github.com/contiv/contivctl
+- name: pull dns container image
+  docker:
```

Comment:
If all use cases for netplugin reuquire dns container, it is perhaps better to pull it in ansible.
Starting can be done by shell, because of the ansible variables it might need. Arguably if no runtime params are needed to be passed, we can run it as part of ansible too. That is an infra service that must be started alongside rest of the things (like netplugin), etc.